gradle/libs.versions.toml

```diff
@@ -353,9 +348,6 @@ managed-micronaut-toml = { module = "io.micronaut.toml:micronaut-toml", version.
 managed-micronaut-tracing-legacy = { module = "io.micronaut:micronaut-tracing", version.ref = "managed-micronaut-tracing-legacy" }
 managed-micronaut-xml = { module = "io.micronaut.xml:micronaut-jackson-xml", version.ref = "managed-micronaut-xml" }
 
-managed-paho-v3 = { module = "org.eclipse.paho:org.eclipse.paho.client.mqttv3", version.ref = "managed-paho-v3" }
-managed-paho-v5 = { module = "org.eclipse.paho:org.eclipse.paho.mqttv5.client", version.ref = "managed-paho-v5" }
-
```

Comment:
I *think* removing these is correct, but I'm unsure 😔

Reply:
I assume it is correct since they are now in the MQTT BOM.